Makefile.one

```diff
@@ -1,20 +1,21 @@
-.PHONY: all
-all:
-	$(MAKE) env
-	$(MAKE) reqs
-	$(MAKE) tracee-ebpf tracee-rules rules
+.PHONY: all | env
```

Comment:
What if you split "clean" and "all" into 2 different make commands ? (curious). I think when you run both at the same cmdline.. it will run both in parallel and clean will interfere with all (needs addressing, just wanted to confirm).

Reply:
I'm getting the same error when

```
make -f Makefile.one clean
make -f Makefile.one all
sudo ./dist/tracee-ebpf
021/12/21 15:34:23 error creating Tracee: validation error: nil bpf object in memory
```

Reply:
@danielpacak what is your environment ? kernel and distro ? were you using non-CORE by default ? If that is the case, then you can run "make -f Makefile.one clean" and then "make -f Makefile.one install-bpf-nocore". This will install the non CO-RE eBPF object into "/tmp/tracee" and allow you to run.

Reply:
That is exactly what this PR is about, to remove the non CO-RE build from tracee-ebpf run. You need to install/create it yourself. You saw the CO-RE eBPF object being built and embedded, but looks like your system isn't using it.

There is also a Makefile I added that creates a tracee-nocore container... it will always compile the eBPF non CORE object in the beginning of the run. There are comments (that I should add to documentation) teaching how to use those files.

Reply:
Got it. Indeed it works. I also cleaned up my /tmp/tracee directory. The environment I used for test is based off of our Vagrantfile (Ubuntu 20.04 / 5.4). The error message was way too confusing "error creating Tracee: validation error: nil bpf object in memory". Is it possible to print more descriptive message in such case?